bin/util.py

```diff
@@ -550,18 +559,32 @@ def strip_newline(s):
     else:
         return s
 
+# check if windows supports symlinks
+if is_windows() or True:
```

Comment:
can you make this lazy and only do it the first time `ensure_symlink` is called? (And remove the `or True`)

Reply:
In that case you could even just try-except the actual link to be made

Reply:
I think its cleaner to always request the developer mode right at startup (and be more annoying to get more people to enable it)
We could also just enable it ourselves but that feels weird, is likely hacky, and might need changes from time to time...